### ISSUE TITLE
What dwc term to use to indicate animal is domesticated?

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/dwc-for-biologging#37]
In camera trap data, researchers often wants to differentiate between wild and domesticated animals. The Camera Trap Data Package schema has a field `is_domesticated` to indicate just that (while annotating). I was curious how one would express this information in Darwin Core.

So far, I'm simply thinking ~~`individualRemarks: domesticated`~~, `organismRemarks: domesticated`. Suggestions welcome.

**Comment by PietrH** [comment]
I agree that a remarks field is probably the best choice for now. I quick search on GBIF shows the term domesticated showing up in occurrenceRemarks, altough I think it's much more bound to the individual than to the occurrence. 

Is `individualRemarks` a dwc field? I can't find it on this list: https://dwc.tdwg.org/list/

I know domesticated organisms are sometimes considered subspecies taxa, in which case it could also be under the Taxon class as a `taxonRemarks`. Or on a less taxonomic level, as an `organismRemarks`

I'd be interested to hear your opinion on this.

**Comment by debpaul** [comment]
(interesting @peterdesmet. you write: <individualRemarks: domesticated> I wonder how parallel this is to dwc:establishmentMeans concept work done by Quentin Groom and what he might offer in the way of potential fields or controlled vocab suggestions). But, I'm stepping in from way outside this group, if the idea is way off, please just ignore.

**Comment by peterdesmet** [comment]
@PietrH sorry, I meant organismRemarks, not individualRemarks

@debpaul yes, I’m part of that publication too 😊. Should look if any of those vocabs are good at capturing this.

**Comment by peterdesmet** [comment]
@debpaul the only one that comes close would be “degreeOfEstablishment = cultivated” @qgroom, would you use this to indicate that the observed animal was domesticated?

**Comment by albenson-usgs** [comment]
Do we know what iNaturalist does for this?

**Comment by peterdesmet** [comment]
No, iNat observations that are captive are considered casual and don't appear on GBIF:

- Original: https://www.inaturalist.org/observations/63270259
- Not on GBIF: https://www.gbif.org/occurrence/search?catalog_number=63270259&dataset_key=50c9509d-22c7-4a22-a47d-8c48425ef4a7&advanced=1&occurrence_status=present

For other iNat obs `establishmentMeans` is set to `wild`: https://www.gbif.org/occurrence/2981047108

**Comment by sarahcd** [comment]
Would be interested to see other examples and any controlled lists on this topic. I've run into questions regarding cases like semi-feral/stray animals, and livestock that are in captivity at some times and largely free-roaming at others. Would these all be classified as domesticated?

**Comment by qgroom** [comment]
There is obviously a difference between a domesticated animal and a domesticated species.
`degreeOfEstablishment` could be used to indicate an animal or plant is in a domesticated situation, but not that an animal is a domesticated species. Therefore, I don't see how it could be used with camera trap data unless the camera was sited in a cage or enclosed field where there were domesticated animals. If you see a cat or dog on a camera trap you don't have much evidence to decide its `degreeOfEstablishment`.

So going back to the original question. I think the answer is that it depends. Does the field `is_domesticated `refer to the taxon or the individual animal?

**Comment by peterdesmet** [comment]
@qgroom it refers to the observed **animal** in this case. It is an assessment of the annotator (i.e this is a domesticated cat, dog, horse, cow) rather than a wild one.

**Comment by qgroom** [comment]
> @qgroom it refers to the observed **animal** in this case. It is an assessment of the annotator, e.g. this is a domesticated cat, dog, horse, cow, rather than a wild one.

That's good, as long as the assessor doesn't assign the status to all potentially domestic animals. It is particularly difficult to know if a cat is feral or wild, but because the latter is likely to have been neutered it makes a big difference.

**Comment by peterdesmet** [comment]
@qgroom knowing the above, what field would you then recommend?

**Comment by qgroom** [comment]
`degreeOfEstablishment` is fine where it is clear to the assessor what they are assessing.
In the case of feral cats and dogs then it might be better to use `dwc:establishmentMeans` to say that they are an introduced species.

**Comment by peterdesmet** [comment]
It is a generic mapping for all camera trap data, so there isn't too much information except:

> `true` if the observed individual(s) are domesticated.

So it might be a dog on a leash (`degreeOfEstablishment: cultivated`) or a domesticated cat or rock pigeon that turned feral and is breeding (`degreeOfEstablishment: reproducing`). It's hard to tell the difference between the two from the `domesticated` flag, so I think the only thing we can say is `establishmentMeans: introduced`, as a:

> Statement about whether an organism or organisms have been introduced to a given place and time through the direct or indirect activity of modern humans.

@qgroom agree?

**Comment by PietrH** [comment]
> So going back to the original question. I think the answer is that it depends. Does the field `is_domesticated `refer to the taxon or the individual animal?

@qgroom , To my understanding domestication is a process that takes multiple generations that inflicts a change into the genetic code and behavior and appearance of the species. If this results in a different taxon/subspecies is a matter of taxon-concept. But As I see it a feral goat dependent from escaped individuals during the great famine, is still domesticated, and an elephant who's ancestors have been in captivity for generations, is still not domesticated. (example from `Clason, A. T. (1977). Jacht en veeteelt van prehistorie tot middeleeuwen. Fibula-Van Dishoeck.`)

To my understanding domestication inflicts large changes in the phenotype and behavior of the animal, that are hereditary. Al tough the exact definition is much more complex: [10.1073/pnas.1501711112](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4371924/)

So I would say domestication as a status is a property of a taxon. This is similar as to how it's handled in iNaturalist where a research grade (so 'wild') observation of a domesticated pigeon is possible (and has a different taxonID from rock pigeon).

**Comment by qgroom** [comment]
@PietrH True, that's why I asked the question. It seems as though they are not using the word domesticated in this literal genetic sense, but something more general that can be applied to a single observation.

**Comment by qgroom** [comment]
BTW: is the "Camera Trap Data Package schema" actually an official standard?

**Comment by PietrH** [comment]
What constitutes a standard as official? TDWG recognition?

**Comment by qgroom** [comment]
Well perhaps "official" is not the right word. But any commonly used data format for camera trap data would be interesting

**Comment by peterdesmet** [comment]
Hi all, thanks for the input. We opted to drop the `is domesticated` field altogether and make use of subspecies to identify domesticated species, e.g. `Canis lupus familiaris` for dog.

@qgroom yes, Camera Trap Data Package schema is intended to be a commonly used data format for camera trap data! More info at https://gitlab.com/oscf/camtrap-dp Which is why I included a generic transformation from that format to Darwin Core in this dwc-for-biologging repository. I have just used it to publish our first camera trap dataset to GBIF: https://doi.org/10.15468/5tb6ze
